eHealthAfrica/aether
# Aet 40 entity validation

#### What does this PR do?
Validates the payload of an entity submitted directly against its assigned schema using avro-python3

#### How to test
Create or update an entity with an invalid payload, that should fail a validation check

#### Related JIRA ticket
[AET-40](https://jira.ehealthafrica.org/browse/AET-40)
